scan-car/src/values/colors.js

```diff
@@ -0,0 +1,68 @@
+/**------------ScanCar Palette Colors ------------ */
+
+/**Based on the custom defined colors ReactJS by Sujar Kumar, June 29th 2020
+https://medium.com/@suraj27/global-colors-in-reactjs-custom-defined-3d4b76efa4fc
+
+To use the colors by name, you need to...
+
+1..... go into the file you want to use the custom colors in
+
+2..... add the following to the imports:
+    import { COLORS } from "src/values/colors.js"
+    .. or whatever path you have to the file from where your component file is located
+
+3.... use COLORS.Color000 to call the color you need. Example:
+    COLORS.Blue300 
+    .. this will grab the string "#4B7DB9" and insert it at that point.
+
+*/
+
+/** */
+
+export const COLORS = {
+    Blue700: "#20334B",
+    Blue600: "#31465F",
+    Blue500: "#355C8A",
+    Blue400: "#3D6A9E",
+    Blue300: "#4B7DB9",
+    Blue200: "#9FBADB",
+    Blue100: "#F0F4F9",
+    Red700: "#3F2222",
+    Red600: "#642B2B",
+    Red500: "#972B2B",
+    Red400: "#D32828",
+    Red300: "#F95959",
+    Red200: "#FA9E9E",
+    Red100: "#FED8D8",
+    Yellow700: "#5A3911",
+    Yellow600: "#774E1D",
+    Yellow500: "#DD933C",
+    Yellow400: "#FDBB6D",
+    Yellow300: "#FDCE95",
+    Yellow200: "#FAE3C7",
+    Yellow100: "#FFF1E0",
+    Purple700: "#413F6B",
+    Purple600: "#3F39AF",
+    Purple500: "#6A62F2",
+    Purple400: "#8882F4",
+    Purple300: "#B1ADF4",
+    Purple200: "#D5D3F6",
+    Purple100: "#F2F1FC",
+    Green700: "#334646",
+    Green600: "#3F6867",
+    Green500: "#36817F",
+    Green400: "#449794",
+    Green300: "#89C1BF",
+    Green200: "#CEEDEC",
+    Green100: "#E0F8F7",
+    Gray700: "#242E38",
+    Gray600: "#3C4249",
+    Gray500: "#60707B",
+    Gray400: "#738896",
+    Gray300: "#A2B6C3",
+    Gray200: "#D1DAE0",
+    Gray100: "#EAEEF1",
+    Gray000: "#F7F7F9",
+    White: "#FFF",
+    Black: "#000"
+}
```

Comment:
Good idea to define the colors by names rather than numbers. See earlier comments about using CSS-variables or consider using Styled Components. - Then you can just use the definitions from this file. :)